new/PRS14131LabetoGasso.xml

```diff
@@ -0,0 +1,57 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14131LabetoGasso" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Labeto Gāššo</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-10-11">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="2">
+                    <persName xml:lang="gez" xml:id="n1">ላቤቶ፡ ጋሾ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Labeto Gāššo</persName>
```

Comment:
might Lābeto be the title ʾabeto?

Reply:
The character in addition 24 on f. 85va looks very much like a ላ - also in comparison with other ለ ሊ ል in this addition, but not like a አ. It may be, that the reading is la-ʾabeto > lā-beto with the preposition la- merged with the initial aleph to lā - whatever may be the meaning or function of la- in this context.
In that case, I keep  ላቤቶ፡ ጋሾ፡ in the ms-record, but I will change to ʾabeto Gāššo in the PRS record and encode ʾabeto as a title.

Reply:
It is absolutely normal in Amharic to merge prepositions and the word beginning. Labeto routinely stands for l-ābeto (not lā-beto!).

Reply:
OK.